src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
+     :active (apply + (map count-map ["Submitted" "Running" "Aborting"]))
+     :terminal (apply + (map count-map ["Failed" "Succeeded" "Aborted"]))}))
+
+(react/defc GlobalSubmissionStatus
+  {:render
+   (fn [{:keys [state]}]
+     (let [status-error (:status-error @state)
+           status-counts (:status-counts @state)]
+       (cond
+         status-error [:span {} [:br {}] "Submissions: " status-error]
+         status-counts [:span {} [:br {}] "Submissions: " (:queued status-counts) " Queued; " (:active status-counts) " Active"]
```

Comment:
@davidangb, @MattPutnam - So, without the `br` tag, the status text and the user name butt up against each other in an ugly way, which is why I had that in there in the first place. This version is slightly cleaner, html-wise.